source/_integrations/binary_sensor.markdown

```diff
@@ -46,6 +46,7 @@ The full list of supported binary sensor device classes is below
 - **None**: Generic on/off. This is the default and doesn't need to be set.
 - **battery**: `on` means low, `off` means normal
 - **battery_charging**: `on` means charging, `off` means not charging
+- **carbon_monoxide**: `on` means carbon monoxide detected, `off` no carbon monoxide (clear)
```

Comment:
And the carbon dioxide?

Reply:
In the architecture discussions, it was agreed that only CO should be added to the binary_sensor device classes.
I have updated the description accordingly